CONTRIBUTING.md

```diff
@@ -1,34 +1,57 @@
-# Contributing to the OCI Cloud Controller Manager
+*Detailed instructions on how to contribute to the project, if applicable. Must include section about Oracle Contributor Agreement with link and instructions*
 
-*Copyright (c) 2017 Oracle and/or its affiliates. All rights reserved.*
+# Contributing to this repository
 
-## Guidelines to raise a PR
+We welcome your contributions! There are multiple ways to contribute.
 
-### Contributor Agreement
+## Opening issues
 
-Pull requests can be made under
-[The Oracle Contributor Agreement](https://www.oracle.com/technetwork/community/oca-486395.html)
-(OCA).
-For pull requests to be accepted, the bottom of
-your commit message must have the following line using your name and
-e-mail address as it appears in the OCA Signatories list.
-```
+For bugs or enhancement requests, please file a GitHub issue unless it's
+security related. When filing a bug remember that the better written the bug is,
+the more likely it is to be fixed. If you think you've found a security
+vulnerability, do not raise a GitHub issue and follow the instructions in our
+[security policy](./SECURITY.md).
+
+## Contributing code
+
+We welcome your code contributions. Before submitting code via a pull request,
+you will need to have signed the [Oracle Contributor Agreement][OCA] (OCA) and
+your commits need to include the following line using the name and e-mail
+address you used to sign the OCA:
+
+```text
 Signed-off-by: Your Name <you@example.org>
 ```
-This can be automatically added to pull requests by committing with:
-```
+
+This can be automatically added to pull requests by committing with `--sign-off`
+or `-s`, e.g.
+
+```text
 git commit --signoff
 ```
-**Only pull requests from committers that can be verified as having
-signed the OCA can be accepted.**
-
-### Commit Message
-* The commits message should prefix "External-ccm:" 
-* All commits should be squashed to a single commit before merging
-
-### Best Practices
```

Comment:
We should keep link to the development guidelines doc. Also, having clean code and E2E results to support the PR is always preferred so we should not leave this out.